src/components/Web3ReactManager/index.tsx

```diff
@@ -21,12 +21,12 @@ export default function Web3ReactManager({ children }: { children: JSX.Element }
   const dispatch = useDispatch()
   /** On user change network from wallet, update chainId in store, only work on EVM wallet */
   useEffect(() => {
-    if (triedEager && chainIdEVM && chainIdState !== chainIdEVM && active && isEVM) {
+    if (triedEager.current && chainIdEVM && chainIdState !== chainIdEVM && active && isEVM) {
```

Comment:
is triedEager a ref? I think it's boolean value, not ref

Reply:
changed to ref now

Reply:
oh, you update the hooks